about-civicactions/diversity-equity-inclusion/resources.md

```diff
@@ -1,17 +1,28 @@
 
 ## DEIA at CivicActions
 
-- **[Neurodiversity: Autism in the Workplace (Trello card w/ attached resources)](https://trello.com/c/K25RohL4/55-neurodiversity-and-eq) (Internal)**
-  - Recording, slide deck, and notes from a DEIA Brownbag call. Learn more about how to support people at work across the spectrum of neurodiversity.
-- **[Personal Pronouns (slide deck)](https://docs.google.com/presentation/d/1v0Ak3oAL5ZrxywQUf1hioUe3BY-73IeaV1XNiAhb9UY/edit#slide=id.g4e8e1b223a_0_50) (Internal)**
-  - What are personal pronouns and why do they matter at CivicActions? In this AHC, the DEIA team introduces pronouns, gender diversity, and how your use of pronouns contributes to team inclusion.
-- **[CivicActions Diversity Survey Results - May 2019 (slide deck)](https://docs.google.com/presentation/d/1QN6tgHgQ6gwyTSAZZ9K5LXqcwPkGt5CbDsbYgyWo9Po/edit#slide=id.gb3711abec_1_2) (Internal)**
-  - Learn about the various kinds of diversity represented on the CivicActions team. The DEIA team uses diversity surveys to guide DEIA and hiring initiatives and understand how we've grown.
-- **[Equity workshop - 2019 Retreat (slide deck)](https://docs.google.com/presentation/d/1QErkPtuT_8ZAbsmhmwsl4tWnckDHlTtOoYdqTwGoNz0/edit#slide=id.g4e8e1b223a_0_50) (Internal)**
-  - What is equity, and how can our organization support a more equitable tech industry? Check out the presentation introducing the equity workshop at the 2019 retreat.
+**[Neurodiversity: Autism in the Workplace (Trello card w/ attached resources)](https://trello.com/c/K25RohL4/55-neurodiversity-and-eq) (Internal)**
+
+Recording, slide deck, and notes from a DEIA Brownbag call. Learn more about how to support people at work across the spectrum of neurodiversity.
+
+**[Personal Pronouns (slide deck)](https://docs.google.com/presentation/d/1v0Ak3oAL5ZrxywQUf1hioUe3BY-73IeaV1XNiAhb9UY/edit#slide=id.g4e8e1b223a_0_50) (Internal)**
+
+What are personal pronouns and why do they matter at CivicActions? In this AHC, the DEIA team introduces pronouns, gender diversity, and how your use of pronouns contributes to team inclusion.
+
+**[CivicActions Diversity Survey Results - May 2019 (slide deck)](https://docs.google.com/presentation/d/1QN6tgHgQ6gwyTSAZZ9K5LXqcwPkGt5CbDsbYgyWo9Po/edit#slide=id.gb3711abec_1_2) (Internal)**
+
+Learn about the various kinds of diversity represented on the CivicActions team. The DEIA team uses diversity surveys to guide DEIA and hiring initiatives and understand how we've grown.
+
+**[Equity workshop - 2019 Retreat (slide deck)](https://docs.google.com/presentation/d/1QErkPtuT_8ZAbsmhmwsl4tWnckDHlTtOoYdqTwGoNz0/edit#slide=id.g4e8e1b223a_0_50) (Internal)**
+
+What is equity, and how can our organization support a more equitable tech industry? Check out the presentation introducing the equity workshop at the 2019 retreat.
+
+\*\*[Code of conduct](https://guidebook.civicactions.com/en/latest/company-policies/code-of-conduct/)
```

Comment:
@ContentBrewmaster can this link be relative instead of absolute i.e. point to the markdown file for code of conduct? Also it missing the `**` at the end to make it bold.